pkg/cmdline/cmdline_other.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build !linux
+// +build !linux
+
+// Package cmdline is parser for kernel command-line args from /proc/cmdline.
+//
+// It's conformant with
```

Comment:
is this comment still true in this file, cmdline_other.go?

Reply:
comment should move, will do, thanks.